src/main/java/com/mraof/minestuck/world/lands/terrain/EndLandType.java

```diff
@@ -46,7 +41,8 @@ public EndLandType()
 	public void registerBlocks(StructureBlockRegistry registry)
 	{
 		registry.setGroundState(Blocks.END_STONE.getDefaultState(), MSFillerBlockTypes.END_STONE);
-		registry.setBlockState("surface", MSBlocks.END_GRASS.getDefaultState());
+		registry.setBlockState("surface", Blocks.END_STONE.getDefaultState());
```

Comment:
Oh, another thing to consider: might ores spawn on the surface with this, and is that acceptable?

Reply:
Coarse end stone is in a strange place right now, agreed. I think it would be preferable if it was the filler block of the world and that the underground was made of it, with a thin layer of end stone and then the end grass blocks. But this would mean modifying the end ore textures so it might be worth debating. If the underground were made of coarse end stone, it would get rid of the issue you are describing with the potentially visible-from-the-surface-ore

Reply:
Personally I don't consider end stone ore -> coarse end stone ore switch to be a big deal, but I suppose you could check what other people think as well.

Reply:
Ground filler block is now coarse end stone, the ore textures have been made to match